### ISSUE TITLE
Running from Agent

### ISSUE THREAD

**Issue by Zhunya** [erikdarlingdata/DarlingData#65]
**Version of the script**

SELECT @version = '1.5', @version_date = '20200501';

**What is the current behavior?**

First, run:
```sql
sp_HumanEvents
      @event_type = 'query'
    , @query_duration_ms = 100
    , @client_hostname = 'VPS'
    , @database_name = 'V2_GATE'
    , @sample_divisor = 1
    , @object_schema = 'Web'
    , @keep_alive = 1
```

Then create job with step code below

```t-sql
sp_HumanEvents
      @output_database_name = 'dbaTools'
    , @output_schema_name = 'xeWeb'
```

Then I do *first* test run with code above (job step) in SSMS

Output
```sql
Do we skip to the GOTO and log tables?
Skipping all the other stuff and going to data logging
Starting data collection.
Sessions without tables found, starting loop.
While, while, while...
Updating #human_events_worker to set is_table_created for keeper_HumanEvents_query
Setting next id after 1 out of 1 total
Found views to create, beginning!
#view_check doesn't exist, creating and populating
Updating #view_check with output database (dbaTools) and schema (xeWeb)
Updating #view_check with table names
Starting view creation loop
creating view humanevents_queries
Setting next id after 3 out of 3 total
Sessions that need data found, starting loop.
Generating insert table statement for keeper_HumanEvents_query
Setting next id after 1 out of 1 total
Found views to create, beginning!
Sessions that need data found, starting loop.
Generating insert table statement for keeper_HumanEvents_query
Setting next id after 1 out of 1 total
Query was canceled by user.
```

For the second attempt got this error:
```sql
Updating #human_events_worker to set is_table_created for keeper_HumanEvents_query
Setting next id after 1 out of 1 total
Found views to create, beginning!
#view_check doesn't exist, creating and populating
Updating #view_check with output database (dbaTools) and schema (xeWeb)
Updating #view_check with table names
Starting view creation loop
creating view humanevents_queries
Msg 50000, Level 16, State 1, Procedure sp_HumanEvents, Line 3110 [Batch Start Line 11]
Error number 2714 with severity 16 and a state of 3 in procedure HumanEvents_Queries on line 1
There is already an object named 'HumanEvents_Queries' in the database.
Msg 2714, Level 16, State 3, Procedure HumanEvents_Queries, Line 1 [Batch Start Line 11]
There is already an object named 'HumanEvents_Queries' in the database.
```

**What is the expected behavior?**

That just works =) 

**Which versions of SQL Server and which OS are affected by this issue? Did this work in previous versions of our procedures?**

**Comment by Zhunya** [comment]
Debugging some:

Added variable printing and got this:

```sql
#view_check doesn't exist, creating and populating
Updating #view_check with output database (dbaTools) and schema (xeWeb)
Updating #view_check with table names
Starting view creation loop
@event_type_check [humanevents_queries], @object_name_check [[dbaTools].[xeWeb].HumanEvents_Queries], database [dbaTools], schema [xeWeb]
@view_sql [CREATE VIEW [dbo].[HumanEvents_Queries]
AS
    WITH query_agg AS 
             (
                SELECT q.query_plan_hash_signed,
                       q.query_hash_signed,
                       q.plan_handle,
                       ISNULL(q.cpu_ms, 0.) AS total_cpu_ms,
```

**Comment by erikdarlingdata** [comment]
Closed via #66
